app/src/components/fields/MultiAutocompleteFieldVariableSize.tsx

```diff
@@ -111,12 +129,50 @@ const MultiAutocompleteFieldVariableSize: React.FC<IMultiAutocompleteField> = (p
 
   const { values, touched, errors, setFieldValue } = useFormikContext<IMultiAutocompleteFieldOption>();
 
+  const [inputValue, setInputValue] = useState('');
+  const [options, setOptions] = useState(props.options || []);
+
+  useEffect(() => {
+    const getOptions = async () => {
+      if (props.type === 'api-search') {
+        const response = await props.getInitList(get(values, props.id));
+        setOptions(response);
+      }
+    };
+
+    getOptions();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  useEffect(() => {
+    const searchSpecies = async () => {
+      if (props.type === 'api-search') {
+        const exsistingValues = get(values, props.id);
```

Comment:
Can we add a type to existingValues? The `get` call returns an `any` which we cant help, but if we know what type it should be returning, lets assign it to existingValues.